src/external/spglib.jl

```diff
@@ -1,15 +1,31 @@
-using PyCall
 # Routines for interaction with spglib
+const SPGLIB = spglib_jll.libsymspg
 
+function spglib_get_error_message()
+    error_code = ccall((:spg_get_error_code, SPGLIB), Cint,(Cvoid,),nothing)
+    return ccall((:spg_get_error_message, SPGLIB), String,(Cint,), error_code)
+end
+
+function spglib_get_symmetry_(unit_cell; maxsize = 52, symprec = 1e-5)
+    rotations    = Array{Cint}(undef, 3, 3, maxsize)
+    translations = Array{Cdouble}(undef, 3, maxsize)
+    cell, positions, indices = unit_cell
+    
+    num_ops = ccall((:spg_get_symmetry, SPGLIB), Cint,
+      (Ptr{Cint}, Ptr{Cdouble}, Cint, Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cdouble),
+       rotations, translations, Cint(maxsize), cell, positions, indices, Cint(length(indices)), symprec)
+    #Note: spglib_jll returns transposed rotation matrix vs the python spglib interface
+    return num_ops, [Vec3(translations[:, i]) for i in 1:num_ops], [Mat3(rotations[:, :, i])' for i in 1:num_ops]
```

Comment:
Use Vec3{Float64} and Mat3{Int} to ensure the c-types are converted to Julia types. Also this lines is too long.